### DISCUSSION TITLE
Custom icons for points in Map panel

### DISCUSSION THREAD

**Discussion by mikelgarciafonseca** [foxglove/community#981] [Q&A]
Hello, is there any way to change the default marker icon used in a lat/lon point on the Map panel?

**Comment by defunctzombie** [comment]
Are you referring to the dot?

**Reply by mikelgarciafonseca** [reply]
I'm refering to the marker icon used in a default GeoJSON point message. But if it is possible to change the default dot for a custom icon I would be interested as well.

Thanks in advance!

**Reply by defunctzombie** [reply]
I don't believe there is a way we have to change the marker icon used in the GeoJSON message. The only thing we offer is changing the color for the dots.
